google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -171,6 +171,14 @@ def _check_unicode_returns(self, connection, additional_tests=None):
         """Ensure requests are returning Unicode responses."""
         return True
 
+    def _get_default_schema_name(self, _):
+        """Get default Cloud Spanner schema name.
+
+        Returns:
+            str: Schema name.
+        """
+        return "unnamed"
```

Comment:
From [the docs](https://cloud.google.com/spanner/docs/information-schema#schemata):

> The name of the schema. An empty string if unnamed.

Does SQLAlchemy support an empty string schema name?

Reply:
I never saw any restrictions like that. I don't actually think SQLAlchemy API by itself uses schema names. Pushing an empty string.